lib/gs-plugin-loader.c

```diff
@@ -1732,100 +1732,6 @@ gs_plugin_loader_pending_apps_remove (GsPluginLoader *plugin_loader,
 	g_idle_add (emit_pending_apps_idle, g_object_ref (plugin_loader));
 }
 
```

Comment:
Similar to my comment in https://github.com/endlessm/gnome-software/pull/331/commits/3bf9c418, I think this commit should also go at the beginning of the rebase branch, right after the tip of the gnome-3-26 branch, since it's also modifying pure upstream code (and hopefully will be dropped for the next rebase).

Reply:
This one I don't agree, because this is a temporary measure, so I prefer to keep it closer to the most recent changes, since it may be changed again soon.